packages/aws-cdk/lib/api/aws-auth/sdk_ini_file.ts

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Not ideal that these were moved around as part of this gigantic PR. I am assuming they were not changed, but it's impossible to tell... If you are willing to revert these moves, it will make it easier to ensure that things wouldn't break unexpectedly.

Reply:
I know, sorry. To aid in building trust in the changes though I've added tests (which we didn't use to have). I would hope those are sufficient.